tests/e2e/env/setup.sh

```diff
@@ -161,6 +161,12 @@ cli wp wc --user=admin tool run install_pages
 echo "Importing some sample data..."
 cli wp import wp-content/plugins/woocommerce/sample-data/sample_products.xml --authors=skip
 
+echo "Adding customer account ..."
+cli wp user create customer customer@woocommercecoree2etestsuite.com --role=customer --user_pass=password
```

Comment:
Seeing the following error when trying to setup locally, I guess we'll have to add the deletion step like the original function, as mentioned in my other comment.

<img width="812" alt="Screenshot 2021-12-21 at 8 08 13 PM" src="https://user-images.githubusercontent.com/68396823/146947813-049b760f-7cf7-4e7b-b9f0-a0828e4d5e7e.png">

Reply:
The guest account deletion step shows it as an invalid account in CI, do we need that since it'll be at the setup level?

<img width="678" alt="Screenshot 2021-12-21 at 8 11 30 PM" src="https://user-images.githubusercontent.com/68396823/146948410-ce9df1b4-7801-4673-8545-fbc944ed27a8.png">

Reply:
@deepakpathania Did you reset the existing local env? Or ran the setup script on top of existing setup?

Reply:
I did reset and setup but maybe the reset failed the first time. 🤔 
Tried resetting and setting up again and got only the CI error of invalid user id, not the first one, but we should still handle this case imo.

Reply:
@deepakpathania Yeah for the guest account the warning appears because we don't create a guest account. I've kept this because it was already present on the Jest setup.